content/guides/getting-started/writing-your-first-component-test.md

```diff
@@ -29,6 +29,31 @@ the following folders to get started:
 - Vue version:
   [guide-getting-started-vue](https://github.com/cypress-io/cypress-component-testing-examples/tree/main/guide-getting-started-vue)
 
+If you want to use your own project, follow the steps in the
```

Comment:
Perhaps this should be in its own box, so that it's more obvious

Reply:
I took the content out of the alert box and put it under its own section called "Getting Started". I think people will be less likely to skip it that way and it felt awkward having three alert boxes before you got to any content. lmkwyt.